tests/base_test.py

```diff
@@ -13,26 +14,24 @@
 '''
 class ArkoudaTest(unittest.TestCase):
 
-    verbose = bool(os.getenv('ARKOUDA_VERBOSE', 'False'))
-    port = os.getenv('ARKOUDA_SERVER_PORT')
-    server = os.getenv('ARKOUDA_SERVER_HOST')
-
-    port_server_set = port != None and server != None
-    full_stack_mode = bool(os.getenv('ARKOUDA_FULL_STACK_TEST', not port_server_set))
+    verbose = True if os.getenv('ARKOUDA_VERBOSE') == 'True' else False
+    port = int(os.getenv('ARKOUDA_SERVER_PORT', 5555))
+    server = os.getenv('ARKOUDA_SERVER_HOST', 'localhost')
+    full_stack_mode = True if os.getenv('ARKOUDA_FULL_STACK_TEST') == 'True' else False
     
     @classmethod
     def setUpClass(cls):
         '''
-        If in full stack mode, Configures and starts the arkouda_server process
+        If in full stack mode, Configures and starts the arkouda_server process, noop otherwise
         
         :return: None
         :raise: RuntimeError if there is an error in configuring or starting arkouda_server
         '''
         if ArkoudaTest.full_stack_mode:
             print('starting in full stack mode')
             try: 
-                nl = get_arkouda_numlocales()
-                (ArkoudaTest.server, ArkoudaTest.port, ArkoudaTest.ak_server) = start_arkouda_server(nl)
+                nl = int(get_arkouda_numlocales())
+                start_arkouda_server(numlocales=nl, host=ArkoudaTest.server, port=ArkoudaTest.port)
```

Comment:
FYI this breaks our nightly XC and CS testing. The original motivation for start_arkouda_server is to launch the server, and once the server is up and running have it report the hostname and port back via a file, since on clusters/supercomputers you don't know the host ahead of time. e.g. if I'm running on cori, you'll only know the server hostname once you've made it through the slurm queue and been given your nodes. 

I think you could just stop passing the server to start_arkouda_server -- I'm not sure when you would need that in full stack mode.

Reply:
@ronawho oh wow, okay, I can take this out. I put it in there because I am pretty sure that specifying the server was being added to Chapel and I wanted to enable configuration of the host (hostname or IP address) for situations where the are multiple NICs for the host.

Reply:
@ronawho okay, made it so the host can be none. Is passing in the ServerPort param okay:

    `cmd = [get_arkouda_server(),
           '--logging={}'.format('true' if log else 'false'),
           '--v={}'.format('true' if verbose else 'false'),
           '--serverConnectionInfo={}'.format(connection_file),
           '-nl {}'.format(numlocales), '--ServerPort={}'.format(port)]`

Please confirm and I'll wrap up these updates and submit a pull request.

Reply:
Think I'm missing something -- looks like that's the same code that's on master

Reply:
@ronawho yes, it was merged into the master yesterday.

Reply:
ooh, sorry -- I misread, I thought you were asking me if this new code snippet looked ok, but you were just asking if the code on master that passes the port is ok. 

Yeah, we don't do anything with the port, that change is good.